optaplanner-quarkus-integration/optaplanner-quarkus/deployment/pom.xml

```diff
@@ -25,6 +25,14 @@
     <dependency>
       <groupId>org.optaplanner</groupId>
       <artifactId>optaplanner-quarkus</artifactId>
+      <exclusions>
+        <!-- The constraints.drl requires this dependency to be excluded, otherwise the quarkus-maven-plugin
+             downloads it at runtime. -->
+        <exclusion>
```

Comment:
This solves the issue with `drools-core-static` getting on the classpath. A user still has to exclude this dependency from the dependency on `optaplanner-quarkus` and `optaplanner-quarkus-json`, etc.

Reply:
Intresting, let's dicuss indeed

Reply:
As discussed, we won't do this change and remove the fail fast if static is not in the classpath with scoreDRL.